### DISCUSSION TITLE
The samples at samples/Robots.Samples.Grasshopper fail to load in Grasshopper.

### DISCUSSION THREAD

**Discussion by tajtac** [visose/Robots#299] [Q&A] [answered]
Hi everyone, I am new to Rhino and Robots. I am trying to learn how to use this module, but when I try to open any of the samples in Grasshopper it says "IO Read error: Unknown data type encountered." Does anyone know a workaround for this? I tried installing previous versions of the Robots plugin (including 1.1.4, which was just before the latest commit that affected the samples in the path above) but it didn't help.

Also, are there any tutorial videos for this plugin?

**Comment by visose** [comment]
Hi, the samples were created with an older version. When you open them, some of them might show an error that says `Input parameter chunk is missing...` but after closing the error window they should work. You probably are running into a different issue.

**Comment by visose** [comment]
Regarding video tutorials, if you search in Youtube for `grasshopper` and `robots` you will find are tutorials that use this plugin.

**Comment by visose** [comment, marked as answer]
Closing this as likely outdated. The Grasshopper samples in the repository have since been refreshed; please use the current samples from `samples/Robots.Samples.Grasshopper` with the current Robots package and Rhino/Grasshopper version. If a current sample such as `simple-example.gh` still fails with `Unknown data type encountered`, please reopen with the exact Rhino version, Robots version, sample file name, and how the file was downloaded.
